portfoliomobile/index.html

```diff
@@ -0,0 +1,183 @@
+                </div>
+                <span><button class="see-more-btn">See More</button></span> 
+            </div>
+            <div id="section-two-one">
+                <div class="projects-card-three"></div>
+                <div><h3 class="tonic-text">Tonic</h3></div>
+                <div class="display-flex projects-info-flex">
+                    <p class="projects-info projects-info-one">CANOPY</p>
+                    <span class="grey-circle projects-info "></span>
+                    <p class="projects-info projects-info-rest">Back End dev</p>
+                    <span class="grey-circle projects-info"></span>
+                    <p class="projects-info projects-info-rest">2015</p>
+                </div>
+                <div><p class="projects-description-one">A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+                <div class ="display-flex projects-icons">
+                    <P class="boxes">Html</p>
+                    <p class="boxes">Css</p>
+                    <p class="boxes">Javascript</p>
+                </div>
+                <span><button class="see-more-btn">See More</button></span> 
+            </div>
+            <div id="section-two-one">
+                <div class="projects-card-four"></div>
+                <div><h3 class="tonic-text">Multi-Post <br> Stories</h3></div>
+                <div class="display-flex projects-info-flex">
+                    <p class="projects-info projects-info-one">CANOPY</p>
+                    <span class="grey-circle projects-info "></span>
+                    <p class="projects-info projects-info-rest">Back End dev</p>
+                    <span class="grey-circle projects-info"></span>
+                    <p class="projects-info projects-info-rest">2015</p>
+                </div>
+                <div><p class="projects-description-one">A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+                <div class ="display-flex projects-icons">
+                    <li class="boxes">Html</li>
+                    <li class="boxes">Css</li>
+                    <li class="boxes">Javascript</li>
+                </div>
+                <span><button class="see-more-btn">See More</button></span> 
+            </div>
+        </div>
+        <div class id="about-me">
+            <div id="section-one-one"> 
+                <div id="title-box">
+                    <h1 id="my-name">About <br>Me</h1>
+                </div>
+                <div id="description-box">
+                    <p id="description-tex">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+            </div>
+            <section id="section-one-two">
+                <ul id="contact-icons">
```

Comment:
- [ ] Please add `Let's Connect` section. it is used to connect with other part of your profile section.
## your design👇
![letsst](https://user-images.githubusercontent.com/66992901/193122431-d18a7f5f-a85b-46b7-81d0-e08ce2a8b029.PNG)

## Figma design👇
![letsfigma](https://user-images.githubusercontent.com/66992901/193122520-948a5f52-2b62-4274-93e1-87236975caa9.PNG)